libs/dotnet-sdk/FeatureboardHttpClient.cs

```diff
@@ -0,0 +1,50 @@
+using Featureboard.DotnetSdk.Models;
+using Microsoft.Extensions.Logging;
+using System.Net;
+using System.Net.Http.Json;
+
+namespace Featureboard.DotnetSdk;
+
+internal class FeatureboardHttpClient : IFeatureboardHttpClient
+{
+  private readonly IHttpClientFactory _httpClientFactory;
+  private readonly ILogger<FeatureboardHttpClient> _logger;
+
+  public FeatureboardHttpClient(IHttpClientFactory httpClientFactory, ILogger<FeatureboardHttpClient> logger)
+  {
+    _httpClientFactory = httpClientFactory;
```

Comment:
Take dependency on `HttpClient` instead of the factory

Reply:
Change class registration to Scoped lifetime too if doing this so don't end up with a captive dependency